reana_server/rest/workflows.py

```diff
@@ -698,19 +699,21 @@ def start_workflow(workflow_id_or_name):  # noqa
                     [WorkflowStatus.finished, WorkflowStatus.failed]:
                 raise ValueError(
                     "Only finished or failed workflows can be restarted.")
+            workflow = restart_workflow(workflow)
```

Comment:
As I read it, I already think that this will "restart" the workflow immediately. What do you think of changing the method name? currently, the docstring explains what it does, and the name explains why it does it (for restarting). I would use something more along the lines of (it is just a suggestion):

```console
def clone_workflow(workflow):
    """Create a copy of workflow in DB for the restarting process."""
...
```